SoftLayer/CLI/monitor/status.py

```diff
@@ -0,0 +1,82 @@
+"""
+usage: sl monitor [<command>] [<args>...] [options]
+
+Manage Monitoring.
+
+The available commands are:
+    status  Show basic monitoring status of servers
+"""
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer import utils
+
+
+@click.command()
+@click.option('--only_hardware', is_flag=True,
+              help="Show only physical servers")
+@click.option('--only_virtual', is_flag=True,
+              help="Show only virtual servers")
+@environment.pass_env
+def cli(env, only_hardware=False, only_virtual=False):
+    """shows SERVER_PING status of all servers."""
+
+    table = formatting.Table([
+        'id', 'datacenter', 'FQDN', 'IP',
+        'status', 'Type', 'last checked at'
+    ])
+    hw_manager = SoftLayer.HardwareMonitorManager(env.client)
+    vs_manager = SoftLayer.VSMonitorManager(env.client)
+    if only_virtual:
+        hardware = []
+        guest = vs_manager.list_status()
+    elif only_hardware:
+        hardware = hw_manager.list_status()
+        guest = []
+    else:
+        hardware = hw_manager.list_status()
+        guest = vs_manager.list_status()
+
+    results = hardware + guest
+    for server_object in results:
+        server = utils.NestedDict(server_object)
+        for monitor in server['networkMonitors']:
+            try:
+                res = monitor['lastResult']['responseStatus']
+                date = monitor['lastResult']['finishTime']
+                ip_address = monitor['ipAddress']
+                monitor_type = monitor['queryType']['name']
+            except KeyError:
```

Comment:
To be more explicit, maybe it's better to do this instead:

```
if 'lastResult' in monitor:
 ...
else:
...
```

The way it works now, a key error for another value could happen and subsequently hidden.